site/src/components/gradients.md

```diff
@@ -14,6 +15,7 @@ There are two main types of gradients.
 
 1. Gradients displayed with font (`.gradient-text-`).
 2. Gradients for blending colors between elements and contains (`.gradient-border-*`).
```

Comment:
Is `contains` meant to be `containers`?

Reply:
This was already in gradient, it's confusing because of the `gradient-border` name but this isn't part of my work on `gradient-card` it's the pre-existing class from the rebrand.

Reply:
I think @homing1 probably knows that, and is just if you could fix the typo while you're in there